docs/man-pages/condor_vacate_job.rst

```diff
@@ -1,119 +1,119 @@
-      
-
 *condor_vacate_job*
-=====================
+===================
+
+Stop running job(s) and relinquish resources back to the host machine.
 
-vacate jobs in the HTCondor queue from the hosts where they are running
-:index:`condor_vacate_job<single: condor_vacate_job; HTCondor commands>`\ :index:`condor_vacate_job command`
+:index:`condor_vacate_job<double: condor_vacate_job; HTCondor commands>`
 
 Synopsis
 --------
 
-**condor_vacate_job** [**-help | -version** ]
+**condor_vacate_job** [**-help** | **-version**]
 
-**condor_vacate_job** [
-**-pool** *centralmanagerhostname[:portnumber]* |
-**-name** *scheddname* ] | [**-addr** *"<a.b.c.d:port>"*]
-[**-fast** ] *cluster... | cluster.process... | user...* |
-**-constraint** *expression* ...
+**condor_vacate_job** [*OPTIONS*] [*cluster*... | *cluster.proc*... | *user*...]
 
-**condor_vacate_job** [
-**-pool** *centralmanagerhostname[:portnumber]* |
-**-name** *scheddname* ] | [**-addr** *"<a.b.c.d:port>"*]
-[**-fast** ] **-all**
+**condor_vacate_job** [**-debug**] [**-long**] [**-totals**] [**-all**] [**-constraint** *expression*]
+[**-pool** *hostname[:portnumber]* | **-name** *scheddname* | **-addr** *"<a.b.c.d:port>"*] [**-fast**]
 
 Description
 -----------
 
-*condor_vacate_job* finds one or more jobs from the HTCondor job queue
-and vacates them from the host(s) where they are currently running. The
-jobs remain in the job queue and return to the idle state.
-
-A running job running will be sent a soft
-kill signal (SIGTERM by default, or whatever is defined as the
-``SoftKillSig`` in the job ClassAd), and HTCondor will restart the job
-from the beginning somewhere else.
-
-If the **-fast** option is used, the job(s) will be immediately killed.
-
-If the **-name** option is specified, the named *condor_schedd* is
-targeted for processing. If the **-addr** option is used, the
-*condor_schedd* at the given address is targeted for processing.
-Otherwise, the local *condor_schedd* is targeted. The jobs to be
-vacated are identified by one or more job identifiers, as described
-below. For any given job, only the owner of the job or one of the queue
-super users (defined by the :macro:`QUEUE_SUPER_USERS` macro) can vacate the
-job.
-
-Using *condor_vacate_job* on jobs which are not currently running has
-no effect.
+Vacate job(s) from the host machine(s) where they are currently running;
+relinquishing resources back to the Execution Point and returning the job(s)
+back to idle to restart resource matchmaking and execution elsewhere. For
+any given job, only the owner of the job or one of the queue super users
+(defined by the :macro:`QUEUE_SUPER_USERS` macro) can vacate the job.
```

Comment:
Might also want to mention that vacating a job causes the condor accountant to stop charging for it (in terms of priority)